src/components/AliasSetup.js

```diff
@@ -21,26 +21,26 @@ const AliasSetup = () => {
       display="flex"
       direction="column"
     >
-      <Span
+      <SpanV2
         textAlign="center"
         margin="30px 0px 0px 0px"
         color={themes.color}
-        size="16px"
+        fontSize="16px"
         textTransform="none"
-        weight="500"
+        fontWeight="500"
       >
         We’re setting up your channel on the {CHAIN_DETAILS[aliasChainId]?.label?.split(' ')[0]} Alias Network.
-      </Span>{" "}
-      <Span
+      </SpanV2>{" "}
+      <SpanV2
         textAlign="center"
         margin="5px 0px 60px 0px"
         color={themes.color}
-        size="16px"
+        fontSize="16px"
         textTransform="none"
-        weight="500"
+        fontWeight="500"
       >
         This usually takes around 5 minutes.
-      </Span>
+      </SpanV2>
       <Item display="flex" direction="row" align="center" margin="50px 0px">
```

Comment:
We can change the Item in sharedStyling to ItemHV2 or ItemVV2 from sharedStylingV2. See if that can be done.